plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -398,59 +398,70 @@ void GSSettingsDlg::UpdateControls()
 		ShowWindow(GetDlgItem(m_hWnd, IDC_OPENCL_DEVICE), SW_HIDE);
 		ShowWindow(GetDlgItem(m_hWnd, IDC_OPENCL_TEXT), SW_HIDE);
 #endif
-
-		ShowWindow(GetDlgItem(m_hWnd, IDC_LOGZ), dx9 ? SW_SHOW: SW_HIDE);
-		ShowWindow(GetDlgItem(m_hWnd, IDC_FBA), dx9 ? SW_SHOW : SW_HIDE);
-
-		ShowWindow(GetDlgItem(m_hWnd, IDC_ACCURATE_DATE), ogl ? SW_SHOW : SW_HIDE);
-		ShowWindow(GetDlgItem(m_hWnd, IDC_ACCURATE_BLEND_UNIT), ogl ? SW_SHOW : SW_HIDE);
-		ShowWindow(GetDlgItem(m_hWnd, IDC_ACCURATE_BLEND_UNIT_TEXT), ogl ? SW_SHOW : SW_HIDE);
-
-		EnableWindow(GetDlgItem(m_hWnd, IDC_MIPMAP_HW), hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_MIPMAP_HW_TEXT), hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_CRC_LEVEL), hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_LARGE_FB), integer_scaling != 1 && hw);
-		EnableWindow(GetDlgItem(m_hWnd, IDC_CRC_LEVEL_TEXT), hw);
 		EnableWindow(GetDlgItem(m_hWnd, IDC_OPENCL_DEVICE), ocl);
+		// Hardware Renderer Settings:
+		ShowWindow(GetDlgItem(m_hWnd, IDC_HW_RENDER_TEXT), hw ? SW_SHOW : SW_HIDE);
+		// Allow 8-bit textures
+		ShowWindow(GetDlgItem(m_hWnd, IDC_PALTEX), hw ? SW_SHOW : SW_HIDE);
+		// Large framebuffer
+		EnableWindow(GetDlgItem(m_hWnd, IDC_LARGE_FB), hw && integer_scaling != 1);
+		ShowWindow(GetDlgItem(m_hWnd, IDC_LARGE_FB), hw ? SW_SHOW : SW_HIDE);
+		// Upscale multiplier
```

Comment:
I'm not sure the comments have much value. I think it's obvious in most cases. Dunno.